static/app/types/system.tsx

```diff
@@ -125,6 +125,7 @@ export interface Config {
   // Maintain isOnPremise key for backcompat (plugins?).
   isOnPremise: boolean;
   isSelfHosted: boolean;
+  isSingleTenant: boolean;
```

Comment:
Was this already being passed and we just didn't have a type for it?

Reply:
nope, i just added this in the [pr linked above](https://github.com/getsentry/sentry/blob/615eaa8862a89511412c65fc411b939d50f4e1de/src/sentry/web/client_config.py#L164)